RESOURCES.md

```diff
@@ -0,0 +1,43 @@
+| [eiffel-community](https://groups.google.com/g/eiffel-community)                                         | [TC](https://github.com/eiffel-community/community/blob/master/GOVERNANCE.md#technical-committee-members) |
+| [eiffel-tc](https://groups.google.com/g/eiffel-tc)                                                       | [TC](https://github.com/eiffel-community/community/blob/master/GOVERNANCE.md#technical-committee-members) |
+| [eiffel-community-security](mailto:eiffel-community-security@googlegroups.com)                           | [Security Officers](https://github.com/eiffel-community/community/blob/master/GOVERNANCE.md#security-officers) |
+| [eiffel-protocol-maintainers](mailto:eiffel-protocol-maintainers@googlegroups.com)                       | ?                                                                                                         |                       
+| [eiffel-commons-maintainers](mailto:eiffel-commons-maintainers@googlegroups.com)                         | ?                                                                                                         |                         
+| [eiffel-easy2use-repository-maintainers](mailto:eiffel-easy2use-repository-maintainers@googlegroups.com) | ?                                                                                                         |
+| [etos-maintainers](mailto:etos-maintainers@googlegroups.com)                                             | ?                                                                                                         |                                             
+| [eiffel-event-repository-maintainers](mailto:eiffel-event-repository-maintainers@googlegroups.com)       | ?                                                                                                         |       
+| [eiffelevents-sdk-go-maintainers](mailto:eiffelevents-sdk-go-maintainers@googlegroups.com)               | ?                                                                                                         |               
+| [eiffelevents-sdk-dotnet-maintainers](mailto:eiffelevents-sdk-dotnet-maintainers@googlegroups.com)       | ?                                                                                                         |       
+| [eiffel-gerrit-plugin-maintainers](mailto:eiffel-gerrit-plugin-maintainers@googlegroups.com)             | Santosh NC                                                                                                |             
+| [eiffel-goer-maintainers](mailto:eiffel-goer-maintainers@googlegroups.com)                               | ?                                                                                                         |                               

```

Comment:
Added my name since I created the mail list, but it can also be Magnus Bäck.

```suggestion
| [eiffel-goer-maintainers](mailto:eiffel-goer-maintainers@googlegroups.com)                               | Tobias Persson                                                                                                         |                               

```